src/components/CheckoutDetails.vue

```diff
@@ -62,27 +64,30 @@ export default class CheckoutDetails extends Vue {
     h1 {
         font-size: 3rem;
         line-height: 3.625rem;
-        font-weight: 300;
+        font-weight: normal;
         letter-spacing: 0.021em;
         margin: 4rem 4rem 1rem 4rem;
     }
 
     h1 .amount {
-        font-weight: 500;
+        font-weight: bold;
     }
 
     .data {
-        font-size: 2rem;
-        line-height: 1.3;
-        opacity: 0.7;
-        padding: 1rem 4rem;
+        margin: 1rem 4rem;
+    }
+
+    .transaction-section {
+        background: white;
```

Comment:
should constants be in place for all colors?

Reply:
Say, I'd assume the Nimiq style lib would provide constants such as colors to be used in CSS. Same issue in other vue files. It's a pain to change something when hard-coded in each file. :/

Reply:
Actually, further down I saw 
```
            background: var(--nimiq-blue);
            background-image: var(--nimiq-blue-bg);
```
so I'd suggest to extend this approach to really have all colors in constants, including `white`, `#f0f0f0`, `#fafafa`, and `#e5e5e5`.

Reply:
(added note "color" to some of the spots)